src/Telegram.php

```diff
@@ -1285,4 +1284,56 @@ public function getUpdateFilter(): ?callable
     {
         return $this->update_filter;
     }
+
+    /**
+     * Converts the name of a class into the name of a command.
+     *
+     * @param string $class For example FooBarCommand
+     *
+     * @return string for example foo_bar. In case of errors, returns an empty string
+     */
+    protected function classNameToCommandName(string $class): string
+    {
+        if (!preg_match('/^(.+)Command$/', $class, $matches)) {
```

Comment:
I see no advantage for preg_match over substr comparisons. I guess the latter would be more efficient. May not matter, but regex seems overkill here.

Reply:
Code updated